voter/models.py

```diff
@@ -102,6 +103,7 @@ class NCVoter(models.Model):
     class Meta:
         verbose_name = "NC Voter"
         verbose_name_plural = "NC Voters"
+        db_table = 'voter_ncvoter'
```

Comment:
Isn't this what it would default to anyway?

Reply:
I would have thought, but was having trouble getting it to work. maybe that was before my migration reset tho. I'll try again removing it.